pkg/tablebackup/basebackup.go

```diff
@@ -19,8 +19,13 @@ import (
 	"github.com/ikitiki/logical_backup/pkg/utils"
 )
 
+type TableNotFoundError error
```

Comment:
I'd do much simpler:
```var TableNotFoundError = errors.New("table not found")```
we don't need a type here

Reply:
Hm... somehow that didn't occur to me in the first place. Thanks, I think your approach is easier.